lib/Alchemy/Phrasea/Command/Maintenance/LazaretFilesSetSizeCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Alchemy\Phrasea\Command\Maintenance;
+
+use Alchemy\Phrasea\Command\Command;
+use Alchemy\Phrasea\Model\Entities\LazaretFile;
+use Alchemy\Phrasea\Model\Repositories\LazaretFileRepository;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class LazaretFilesSetSizeCommand extends Command
+{
+    public function __construct()
+    {
+        parent::__construct('lazaret:set_sizes');
+
+        $this
+            ->setDescription('Set the null size in the LazaretFiles table')
+            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'dry run, count')
```

Comment:
better use **--dry** like in recent commands

Reply:
ok